tavern_server/src/db_tables.sql

```diff
@@ -0,0 +1,676 @@
+	'Ride', 
+	'Sense Motive', 
+	'Sleight of Hand', 
+	'Spellcraft', 
+	'Stealth', 
+	'Survival', 
+	'Swim', 
+	'Use Magic Device'
+);
+
+CREATE TYPE SaveThrow AS ENUM (
+	'Fortitude', 
+	'Reflex',
+	'Will'
+);
+
+CREATE TYPE CharacterStat AS ENUM (
+	'Name',
+	'Race',
+	'Size',
+	'Height',
+	'Weight',
+	'Age',
+	'Gender',
+	'Alignment',
+	'Deity',
+	'Languages',
+	'Appearance'
+);
+
+CREATE TYPE CombatStat AS ENUM (
+	'Melee Attack Bonus',
+	'Ranged Attack Bonus',
+	'CMB',
+	'CMD',
+	'Armor Class',
+	'Touch AC',
+	'Flat-Footed AC',
+	'Initiative Bonus',
+	'Damage Reduction',
+	'Spell Resistance',
+	'Speed',
+	'Fortititude Save',
+	'Reflex Save',
+	'Will Save'
+);
+
+CREATE TYPE RaceType AS ENUM (
+	'Abberation', 
+	'Animal', 
+	'Construct', 
+	'Dragon', 
+	'Fey', 
+	'Humanoid', 
+	'Magical Beast', 
+	'Monstrous Humanoid', 
+	'Ooze', 
+	'Outsider', 
+	'Plant', 
+	'Undead', 
+	'Vermin'
+);
+
+CREATE TYPE CasterType AS ENUM (
+	'Spontaneous', 
+	'Prepared'
+);
+
+CREATE TYPE MagicSchool AS ENUM (
+	'Abjuration', 
+	'Conjuration', 
+	'Divination', 
+	'Enchantment', 
+	'Evocation', 
+	'Illusion', 
+	'Necromancy', 
+	'Transmutation'
+);
+
+CREATE TYPE SpellRange AS ENUM (
+	'Personal',
+	'Touch',
+	'Close',
+	'Medium',
+	'Long',
+	'Unlimited'
+);
+
+CREATE TYPE WeaponClass AS ENUM (
+	'Axes', 
+	'Heavy Blades', 
+	'Light Blades', 
+	'Bows', 
+	'Close', 
+	'Crossbows', 
+	'Double', 
+	'Firearms', 
+	'Flails', 
+	'Hammers', 
+	'Monk', 
+	'Natural', 
+	'Polearms', 
+	'Siege Engines', 
+	'Spears', 
+	'Thrown', 
+	'Tribal'
+);
+
+CREATE TYPE ArmorClass AS ENUM (
+	'Light', 
+	'Medium', 
+	'Heavy'
+);
+
+CREATE TYPE EquipmentSlot AS ENUM (
+	'None',
+	'Armor', 
+	'Belts', 
+	'Body', 
+	'Chest', 
+	'Eyes', 
+	'Feet', 
+	'Hands', 
+	'Head', 
+	'Headband', 
+	'Neck', 
+	'Ring (left)', 
+	'Ring (right)', 
+	'Shield', 
+	'Shoulders', 
+	'Wrist'
+);
+
+CREATE TYPE DamageType AS ENUM (
+	'Bludgeoning', 
+	'Slashing', 
+	'Piercing', 
+	'Energy', 
+	'Acid', 
+	'Fire', 
+	'Electricity', 
+	'Cold', 
+	'Sonic', 
+	'Positive', 
+	'Negative', 
+	'Nonlethal'
+);
+
+CREATE TYPE Material AS ENUM (
+	'Nonspecific', 
+	'Abysium', 
+	'Adamantine', 
+	'Angelskin', 
+	'Aszite', 
+	'Magic Bridge Basalt', 
+	'Blight Quartz', 
+	'Bone', 
+	'Bronze', 
+	'Bulette', 
+	'Coral', 
+	'Clamshell', 
+	'Chitin', 
+	'Crypstone', 
+	'Blood Crystal',
+	'Darkleaf Cloth',
+	'Darkwood',
+	'Djezet',
+	'Dragonhide',
+	'Dragonskin',
+	'Druchite',
+	'Eel Hide',
+	'Elysian Bronze',
+	'Gold',
+	'Greenwood',
+	'Griffon Mane',
+	'Horacalcum',
+	'Inubrix',
+	'Cold Iron',
+	'Mindglass',
+	'Mithral',
+	'Noqual',
+	'Obsidian',
+	'Serpentstone',
+	'Siccatite',
+	'Alchemical Silver',
+	'Silversheen',
+	'Skymetal Alloy',
+	'Spiresteel',
+	'Fire-forged Steel',
+	'Frost-Forged Steel',
+	'Living Steel',
+	'Singing Steel',
+	'Stone',
+	'Sunsilver',
+	'Viridium',
+	'Voidglass',
+	'Whipwood',
+	'Wyroot'
+);
+
+--table declarations
+--user creds table - also used for hashing info
+CREATE TABLE UserInfo (
```

Comment:
BTW, I would prefer if this were called `Users` rather than `UserInfo`. When I took a database class, the general naming rule we went by was that a table is named the plural of the thing stored in it, or a combination of two things for junction tables.

Reply:
Implemented.